resources/views/promobar/components/countdown.blade.php

```diff
@@ -40,6 +86,7 @@
                 <span x-text="timer.minutes">{{ $minutes() }}</span> :
                 <span x-text="timer.seconds">{{ $seconds() }}</span>
             @else
+            here
```

Comment:
@iamkarsoft Should this "here" be removed?

Reply:
Thanks How did that slip into this.